snapshot/git/gitdb/stream.go

```diff
@@ -0,0 +1,66 @@
+package gitdb
+
+import (
+	"fmt"
+
+	"github.com/scootdev/scoot/snapshot"
+)
+
+// A Stream is a sequence of SnapshotWithHistory's that updates via a
+// Git ref that can be fetched from a Git remote
+type StreamConfig struct {
+	// Name (used in IDs, e.g. sm)
+	Name string
+
+	// Remote to fetch from (e.g. https://github.com/scootdev/scoot)
+	Remote string
+
+	// Name of ref to follow in data repo (e.g. refs/remotes/upstream/master)
+	RefSpec string
```

Comment:
Don't need it for this iteration; happy to remove.

We'll use it in uploading to find merge-base for bundle basis selection.